src/components/tile-details-panel/tile-metadata.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import styled from "styled-components";
+import { Tile3D } from "@loaders.gl/tiles";
+import { getChildrenInfo } from "../../utils/debug/tile-debug";
+import {
+  formatFloatNumber,
+  formatIntValue,
+  formatStringValue,
+} from "../../utils/format/format-utils";
+import { getBoundingType } from "../../utils/debug/bounding-volume";
+import { PanelHorizontalLine, Title } from "../common";
+
+const REFINEMENT_TYPES = {
+  1: "Add",
+  2: "Replace",
+};
+
+const TileInfoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 24px;
+  margin: 0 18px;
+`;
+
+const TileInfoTitle = styled(Title)`
+  flex: 1;
+`;
+
+const TileInfoValue = styled(Title)`
+  flex: 1;
+  color: ${({ theme }) => theme.colors.mainDimColorInverted};
+  font-weight: 400;
+
+  //Must be added? added cause lod metric type is a long single word and it does not fit in properly, this is probably because scrollbar is too wide
+  //also oveflow: hidden will help
+  word-break: break-all;
+  white-space: pre-wrap;
+`;
+
+const TileInfoDevider = styled(PanelHorizontalLine)`
+  margin: 11px 16px;
+`;
+
+const NO_DATA = "No Data";
+
+export const TileMetadata = ({ tile }: { tile: Tile3D }) => {
+  const {
+    id,
+    type,
+    header: { children: tileChildren },
+    _distanceToCamera,
+    content: { vertexCount, texture, material },
+    refine,
+    lodMetricType,
+    lodMetricValue,
+    // @ts-expect-error - Property '_screenSpaceError' is private and only accessible within class 'TileHeader'.
+    _screenSpaceError,
```

Comment:
```suggestion
   screenSpaceError,
```
Shoud work. I've added a getter to the class